configuration/dot-env-changes.rst

```diff
@@ -55,17 +55,18 @@ changes can be made to any Symfony 3.4 or higher app:
 #. Update ``.gitignore``:
 
    .. code-block:: diff
+      :dedent: 0
```

Comment:
Is it possible to set this in the config without the need to change all examples?

Reply:
In fact we don't care about this flag, it just helps Sphinx to get the indentation.
Here are 2 identical exemples
```
.. code-block:: diff
    + added
       not changed
```

```
.. code-block:: diff
       added
     + not changed
```

In the first one, Sphinx is able to get that indentation of the bloc is `4`. But in the second, is it `6` like suggested by the first line? or `4`?
So we need something to help Sphinx to figure out the indentation of the bloc.

I would need Sphinx locally to test if this is really needed, maybe you can help me with that?